### PR TITLE
Fixes plone-pat-modal redirection issue after changing state from content menu

### PR DESCRIPTION
**Pull request by rohnsha0**
fixes [#3989](https://github.com/plone/Products.CMFPlone/issues/3989)

### CODE REVIEW

**Comment by rohnsha0** [comment]
pre-commit.ci autofix

**Comment by rohnsha0** [comment]
@jenkins-plone-org please run jobs

**Comment by rohnsha0** [comment]
again same failure 

```
Setup failed:
SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 124
Current browser version is 127.0.6533.99 with binary path /usr/bin/google-chrome
```

**Comment by petschki** [comment]
@jenkins-plone-org please run jobs

File: news/3989.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+Fixes plone-pat-modal redirection issue after changing state from content menu [rohnsha0]
```
Comment (petschki) [inline news/3989.bugfix:1]:
```suggestion
Fixes pat-plone-modal redirection issue after changing state from content menu [rohnsha0]
```

**Review by petschki — CHANGES_REQUESTED** [review]
Just a typo in the changes. otherwise LGTM.

**Comment by rohnsha0** [comment]
> Just a typo in the changes. otherwise LGTM.

done...

**Review by petschki — APPROVED** [review]
Merge when green.

**Comment by rohnsha0** [comment]
@jenkins-plone-org please run jobs

**Pull request merged**